clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupSpec.java

```diff
@@ -420,15 +420,31 @@ public void createServerGroup_InputsEc2WithoutLoadBalacingTest()
         ArgumentCaptor.forClass(RegisterTaskDefinitionRequest.class);
     verify(mockECS).registerTaskDefinition(registerTaskDefArgs.capture());
     RegisterTaskDefinitionRequest seenTaskDefRequest = registerTaskDefArgs.getValue();
-    assertEquals(expectedServerGroupName, seenTaskDefRequest.getFamily());
-    assertEquals(1, seenTaskDefRequest.getContainerDefinitions().size());
+    assertEquals(
+        expectedServerGroupName,
+        seenTaskDefRequest.getFamily(),
+        "Expected server group name is ecs-integInputsEc2NoLoadBalancing. Server group name : "
```

Comment:
(general comment) I like that we're including the expected _and_ actual value in the error msgs, but would prefer if we used `String.format()` and provided them as params. Like:

```
String.format(
    "Expected server group name to be '%s' but saw '%s'", 
    expectedServerGroupName, seenTaskDefRequest.getFamily())
```